### ISSUE TITLE
Why no batch normalization?

### ISSUE THREAD

**Issue by Suyuanhang** [alterzero/RBPN-PyTorch#41]
This is not a bug, but I'm curious that why you do not use batch normalization? This is not discussed in the paper, can you shed some light on this?

**Comment by HikariS97** [comment]
Google EDSR
